data/entity/npc/monster/draconic/dragon.drops.toml

```
@@ -0,0 +1,181 @@
+  { id = "rune_limbs", chance = 40 },
+  { id = "adamant_bar", amount = 2, chance = 24 },
+  { id = "curry", chance = 24 },
+  { table = "rare_drop_table", chance = 16 },
+]
+
+[iron_dragon_tertiary]
+roll = 10000
+drops = [
+  { table = "hard_clue_scroll", chance = 78 },
+  { id = "draconic_visage" },
+]
+
+[steel_dragon_drop_table]
+type = "all"
+drops = [
+  { table = "steel_dragon_primary" },
+  { table = "steel_dragon_secondary" },
+  { table = "steel_dragon_tertiary" },
+]
+
+[steel_dragon_primary]
+drops = [
+  { id = "dragon_bones" },
+  { id = "steel_bar", amount = 5 },
+]
+
+[steel_dragon_secondary]
+roll = 512
+drops = [
+  { id = "dragon_plateskirt" },
+  { id = "dragon_platelegs" },
+  { id = "rune_dart_p", amount = 12, chance = 28 },
+  { id = "rune_mace", chance = 16 },
+  { id = "adamant_kiteshield", chance = 8 },
+  { id = "rune_knife", amount = 7, chance = 12 },
+  { id = "rune_hatchet", chance = 8 },
+  { id = "rune_full_helm", chance = 4 },
+  { id = "blood_rune", amount = 20, chance = 76 },
+  { id = "rune_javelin", amount = 7, chance = 80 },
+  { id = "runite_bolts", min = 2, max = 12, chance = 24 },
+  { id = "soul_rune", amount = 5, chance = 20 },
+  { id = "coins", amount = 470, chance = 68 },
+  { id = "coins", amount = 650, chance = 20 },
+  { id = "super_attack_3", chance = 52 },
+  { id = "rune_limbs", chance = 32 },
+  { id = "rune_bar", chance = 12 },
+  { id = "super_defence_2", chance = 12 },
+  { id = "curry", chance = 4 },
+  { id = "curry", amount = 2, chance = 4 },
+  { table = "rare_drop_table", chance = 16 },
+]
+
+[steel_dragon_tertiary]
+roll = 10000
+drops = [
+  { table = "hard_clue_scroll", chance = 156 },
+  { id = "draconic_visage" },
+]
+
+[red_dragon_drop_table]
+type = "all"
+drops = [
+  { table = "red_dragon_primary" },
+  { table = "red_dragon_secondary" },
+  { table = "hard_clue_scroll", roll = 128 },
+]
+
+[red_dragon_primary]
+drops = [
+  { id = "dragon_bones" },
+  { id = "red_dragonhide" },
+]
+
+[red_dragon_secondary]
+roll = 128
+drops = [
+  { id = "mithril_2h_sword", chance = 4 },
+  { id = "mithril_hatchet", chance = 3 },
+  { id = "mithril_battleaxe", chance = 3 },
+  { id = "rune_dart", amount = 8, chance = 3 },
+  { id = "mithril_javelin", amount = 20 },
+  { id = "mithril_kiteshield" },
+  { id = "adamant_platebody" },
+  { id = "rune_longsword" },
+  { id = "rune_arrow", amount = 4, chance = 8 },
+  { id = "law_rune", amount = 4, chance = 5 },
+  { id = "blood_rune", amount = 2, chance = 4 },
+  { id = "death_rune", amount = 5, chance = 3 },
+  { table = "herb_drop_table", chance = 2 },
+  { id = "coins", amount = 196, chance = 40 },
+  { id = "coins", amount = 66, chance = 29 },
+  { id = "coins", amount = 330, chance = 10 },
+  { id = "coins", amount = 690 },
+  { id = "chocolate_cake", amount = 3, chance = 3 },
+  { id = "adamant_bar" },
+  { table = "gem_drop_table", chance = 5 },
+]
+
+[baby_dragon_drop_table]
+type = "all"
+drops = [
+    { table = "babydragon_bones" }
```

```suggestion
    { id = "babydragon_bones" }
```

Another one, run the tests locally when it's ready to speed up getting PRs through